test/parallel/test-symlinked-peer-modules.js

```diff
@@ -0,0 +1,78 @@
+// Flags: --preserve-symlinks
+/* eslint no-irregular-whitespace: 0 */
```

Comment:
Why is `no-irregular-whitespace: 0` necessary?  Because of the ASCII diagram below?

(It's not really ASCII but you get my meaning.)